easybuild/easyconfigs/c/CNVkit/CNVkit-0.9.6-foss-2019a-Python-3.7.2-R-3.6.0.eb

```diff
@@ -0,0 +1,61 @@
+easyblock = 'PythonBundle'
+
+name = 'CNVkit'
+version = '0.9.6'
+versionsuffix = '-Python-%(pyver)s-R-%(rver)s'
+
+homepage = 'http://github.com/etal/cnvkit'
+description = """A command-line toolkit and Python library for detecting copy
+ number variants and alterations genome-wide from high-throughput sequencing."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('Biopython', '1.73'),
+    ('matplotlib', '3.0.3', '-Python-%(pyver)s'),
+    ('networkx', '2.3', '-Python-%(pyver)s'),
+    ('Pillow', '6.0.0'),
+    ('Pysam', '0.15.2'),
+    ('PyYAML', '5.1'),
+    ('R', '3.6.0'),
+    ('R-bundle-Bioconductor', '3.9', '-R-%(rver)s'),
+]
+
+use_pip = False
+
+exts_list = [
+    ('reportlab', '3.5.23', {
+        'source_urls': ['https://pypi.python.org/packages/source/r/reportlab'],
+        'checksums': ['6c81ee26753fa09062d8404f6340eefb02849608b619e3843e0d17a7cda8798f'],
+    }),
+    ('pyfaidx', '0.5.5.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyfaidx'],
+        'checksums': ['9ac22bdc7b9c5d995d32eb9dc278af9ba970481636ec75c0d687d38c26446caa'],
+    }),
+    ('future', '0.17.1', {
```

Comment:
`future` is already included in Python

Reply:
Totally missed that change in Python. Fixed :+1: